src/components/tooltip/TooltipDirective.js

```diff
@@ -515,22 +562,9 @@ goog.require('ga_topic_service');
                         storeFeature(value.layerBodId, features[i]);
                       }
                     }
-                    var mapSize = map.getSize();
-                    var mapExtent = map.getView().calculateExtent(mapSize);
-                    var htmlUrl = scope.options.htmlUrlTemplate
-                                  .replace('{Topic}', gaTopic.get().id)
-                                  .replace('{Layer}', value.layerBodId)
-                                  .replace('{Feature}', value.featureId);
-                    $http.get(htmlUrl, {
-                      timeout: canceler.promise,
```

Comment:
The `cache:true` is only a user session cache when the user will reopen the page the cache will be empty.